2022-06-20 10:47:46,159 INFO    MainThread:20220 [internal.py:wandb_internal():90] W&B internal server running at pid: 20220, started at: 2022-06-20 10:47:46.158040
2022-06-20 10:47:46,161 INFO    WriterThread:20220 [datastore.py:open_for_write():75] open: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\run-2o6418af.wandb
2022-06-20 10:47:46,164 DEBUG   SenderThread:20220 [sender.py:send():235] send: header
2022-06-20 10:47:46,164 DEBUG   SenderThread:20220 [sender.py:send():235] send: run
2022-06-20 10:47:46,623 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: check_version
2022-06-20 10:47:46,638 INFO    SenderThread:20220 [dir_watcher.py:__init__():205] watching files in: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files
2022-06-20 10:47:46,652 INFO    SenderThread:20220 [sender.py:_start_run_threads():812] run started: 2o6418af with start time 1655714864
2022-06-20 10:47:46,652 DEBUG   SenderThread:20220 [sender.py:send():235] send: summary
2022-06-20 10:47:46,653 INFO    SenderThread:20220 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2022-06-20 10:47:46,653 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: check_version
2022-06-20 10:47:46,888 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: run_start
2022-06-20 10:47:46,917 DEBUG   HandlerThread:20220 [meta.py:__init__():35] meta init
2022-06-20 10:47:46,917 DEBUG   HandlerThread:20220 [meta.py:__init__():49] meta init done
2022-06-20 10:47:46,917 DEBUG   HandlerThread:20220 [meta.py:probe():209] probe
2022-06-20 10:47:47,000 DEBUG   HandlerThread:20220 [meta.py:_setup_git():199] setup git
2022-06-20 10:47:47,131 DEBUG   HandlerThread:20220 [meta.py:_setup_git():206] setup git done
2022-06-20 10:47:47,132 DEBUG   HandlerThread:20220 [meta.py:_save_pip():53] save pip
2022-06-20 10:47:47,134 DEBUG   HandlerThread:20220 [meta.py:_save_pip():67] save pip done
2022-06-20 10:47:47,134 DEBUG   HandlerThread:20220 [meta.py:probe():247] probe done
2022-06-20 10:47:47,345 DEBUG   SenderThread:20220 [sender.py:send():235] send: files
2022-06-20 10:47:47,345 INFO    SenderThread:20220 [sender.py:_save_file():949] saving file wandb-metadata.json with policy now
2022-06-20 10:47:47,345 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:47:47,346 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:47:47,455 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: pause
2022-06-20 10:47:47,455 INFO    HandlerThread:20220 [handler.py:handle_request_pause():695] stopping system metrics thread
2022-06-20 10:47:47,653 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_created():256] file/dir created: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\wandb-metadata.json
2022-06-20 10:47:47,654 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_created():256] file/dir created: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\wandb-summary.json
2022-06-20 10:47:47,654 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_created():256] file/dir created: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\requirements.txt
2022-06-20 10:47:47,655 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_created():256] file/dir created: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\output.log
2022-06-20 10:47:47,709 DEBUG   SenderThread:20220 [sender.py:send():235] send: telemetry
2022-06-20 10:47:47,711 DEBUG   SenderThread:20220 [sender.py:send():235] send: stats
2022-06-20 10:47:48,322 INFO    Thread-17 :20220 [upload_job.py:push():132] Uploaded file C:\Users\jtros\AppData\Local\Temp\tmpair0imfwwandb\3f25hlip-wandb-metadata.json
2022-06-20 10:48:02,716 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:48:02,717 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:48:16,162 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: resume
2022-06-20 10:48:16,162 INFO    HandlerThread:20220 [handler.py:handle_request_resume():686] starting system metrics thread
2022-06-20 10:48:16,168 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: pause
2022-06-20 10:48:16,168 INFO    HandlerThread:20220 [handler.py:handle_request_pause():695] stopping system metrics thread
2022-06-20 10:48:16,901 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_modified():269] file/dir modified: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\config.yaml
2022-06-20 10:48:17,121 DEBUG   SenderThread:20220 [sender.py:send():235] send: stats
2022-06-20 10:48:18,208 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:48:18,210 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:48:33,561 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:48:33,562 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:48:49,373 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:48:49,373 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:49:05,083 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:49:05,084 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:49:21,191 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: stop_status
2022-06-20 10:49:21,191 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: stop_status
2022-06-20 10:49:21,468 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: resume
2022-06-20 10:49:21,468 INFO    HandlerThread:20220 [handler.py:handle_request_resume():686] starting system metrics thread
2022-06-20 10:49:22,025 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: poll_exit
2022-06-20 10:49:22,443 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_modified():269] file/dir modified: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\output.log
2022-06-20 10:49:22,983 DEBUG   SenderThread:20220 [sender.py:send():235] send: telemetry
2022-06-20 10:49:22,984 DEBUG   SenderThread:20220 [sender.py:send():235] send: telemetry
2022-06-20 10:49:22,984 DEBUG   SenderThread:20220 [sender.py:send():235] send: exit
2022-06-20 10:49:22,985 INFO    SenderThread:20220 [sender.py:send_exit():371] handling exit code: 0
2022-06-20 10:49:22,985 INFO    SenderThread:20220 [sender.py:send_exit():373] handling runtime: 2
2022-06-20 10:49:22,987 INFO    SenderThread:20220 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2022-06-20 10:49:22,987 INFO    SenderThread:20220 [sender.py:send_exit():379] send defer
2022-06-20 10:49:22,987 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: poll_exit
2022-06-20 10:49:22,988 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:22,988 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 0
2022-06-20 10:49:22,988 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:22,988 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 0
2022-06-20 10:49:22,988 INFO    SenderThread:20220 [sender.py:transition_state():392] send defer: 1
2022-06-20 10:49:22,989 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:22,989 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 1
2022-06-20 10:49:23,110 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: poll_exit
2022-06-20 10:49:23,111 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:23,111 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 1
2022-06-20 10:49:23,111 INFO    SenderThread:20220 [sender.py:transition_state():392] send defer: 2
2022-06-20 10:49:23,111 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: poll_exit
2022-06-20 10:49:23,112 DEBUG   SenderThread:20220 [sender.py:send():235] send: stats
2022-06-20 10:49:23,112 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:23,112 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 2
2022-06-20 10:49:23,112 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:23,113 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 2
2022-06-20 10:49:23,113 INFO    SenderThread:20220 [sender.py:transition_state():392] send defer: 3
2022-06-20 10:49:23,113 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:23,113 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 3
2022-06-20 10:49:23,113 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:23,113 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 3
2022-06-20 10:49:23,113 INFO    SenderThread:20220 [sender.py:transition_state():392] send defer: 4
2022-06-20 10:49:23,114 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:23,114 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 4
2022-06-20 10:49:23,114 DEBUG   SenderThread:20220 [sender.py:send():235] send: summary
2022-06-20 10:49:23,115 INFO    SenderThread:20220 [sender.py:_save_file():949] saving file wandb-summary.json with policy end
2022-06-20 10:49:23,115 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:23,115 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 4
2022-06-20 10:49:23,115 INFO    SenderThread:20220 [sender.py:transition_state():392] send defer: 5
2022-06-20 10:49:23,116 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: defer
2022-06-20 10:49:23,116 INFO    HandlerThread:20220 [handler.py:handle_request_defer():164] handle defer: 5
2022-06-20 10:49:23,116 DEBUG   SenderThread:20220 [sender.py:send_request():249] send_request: defer
2022-06-20 10:49:23,116 INFO    SenderThread:20220 [sender.py:send_request_defer():388] handle sender defer: 5
2022-06-20 10:49:23,226 DEBUG   HandlerThread:20220 [handler.py:handle_request():141] handle_request: poll_exit
2022-06-20 10:49:23,459 INFO    Thread-11 :20220 [dir_watcher.py:_on_file_modified():269] file/dir modified: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\files\wandb-summary.json
2022-06-20 10:49:23,599 ERROR   SenderThread:20220 [internal_api.py:execute():143] 409 response executing GraphQL.
2022-06-20 10:49:23,600 ERROR   SenderThread:20220 [internal_api.py:execute():144] {"errors":[{"message":"Error 1062: Duplicate entry '1060928-2o6418af' for key 'PRIMARY'","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2022-06-20 10:49:24,118 INFO    WriterThread:20220 [datastore.py:close():279] close: C:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\wandb\run-20220620_104738-2o6418af\run-2o6418af.wandb
2022-06-20 10:49:24,229 INFO    HandlerThread:20220 [handler.py:finish():806] shutting down handler
2022-06-20 10:49:24,229 ERROR   MainThread:20220 [internal.py:wandb_internal():165] Thread SenderThread:
Traceback (most recent call last):
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\apis\normalize.py", line 22, in wrapper
    return func(*args, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\internal_api.py", line 1319, in upsert_run
    response = self.gql(mutation, variable_values=variable_values, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\lib\retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\internal_api.py", line 140, in execute
    return self.client.execute(*args, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\vendor\gql-0.2.0\wandb_gql\transport\requests.py", line 39, in execute
    request.raise_for_status()
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\requests\models.py", line 960, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\internal_util.py", line 51, in run
    self._run()
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\internal_util.py", line 102, in _run
    self._process(record)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\internal.py", line 310, in _process
    self._sm.send(record)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\sender.py", line 237, in send
    send_handler(record)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\sender.py", line 251, in send_request
    send_handler(record)
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\sender.py", line 411, in send_request_defer
    self.debounce()
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\sender.py", line 320, in debounce
    self._debounce_config()
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\sdk\internal\sender.py", line 326, in _debounce_config
    self._api.upsert_run(
  File "c:\Users\jtros\CS\cours\PoleProjet\FormationRecherche\Tsunami\TP\sceance4\Tsunami\venv_tsunami\lib\site-packages\wandb\apis\normalize.py", line 24, in wrapper
    raise CommError(err.response, err)
wandb.errors.CommError: <Response [409]>
2022-06-20 10:49:26,256 INFO    MainThread:20220 [internal.py:handle_exit():80] Internal process exited
